pkg/hsskey/hsskey.go

```diff
@@ -182,6 +189,38 @@ func GetAllHss(warnings io.Writer, verboseDangerous bool, eepromPattern string,
 	return nil, fmt.Errorf("failed all HSS retrieval attempts")
 }
 
+// evaluateHssPaths evaluates a filepath to return the file or contents if a directory.
+// This function evaluates the base pointer for symlinks.
+func evaluateHssPaths(hssFiles string) ([]string, error) {
```

Comment:
I don't remember why we were accepting comma separated list of files, but if that does not make sense any more, this LGTM. 

A unhappy thing is original users later find this out, or this should up as a regression. 

It seems this code can easily accomendate comma separated list of files, or directories, 

```
func evaluateHssPaths(hssFiles string) ([]string, error) {
      if len(hssFiles) == 0 {
          return []string{}, nil
      }
      hssFileSlice := strings.Split(strings.TrimSpace(hssFiles), "," ) // or make delimitter as a constant. 
      hssFileArr := []string{} 
      if len(hssFileSlice) > 1 {
            for _, nextFile := range hssFileSlice {
                  nextArr, nextErr := evaluateHssPaths(nextFile)
                  // may need change return, `[]string, error`, as `[]string, [] error`, and bubble up error.
                  hssFileArr = append(hssFileArr, nextArr)
            }
            return hssFileArr
      }
     
      // same thing as existing, modulo small adjustments 
}
```

Reply:
Yeah, it really wasn't around for long and was unused, but we can keep it since it's trivial. Added splitting back. Thx.